package.json

```diff
@@ -21,7 +21,8 @@
     "cloudant": "^1.4.0",
     "debug": "^2.2.0",
     "lodash": "^3.10.1",
-    "loopback-connector": "^2.3.0"
+    "loopback-connector": "^2.3.0",
+    "strong-globalize": "^2.6.5"
```

Comment:
Caret will pick up 2.6.6 anyways, but I'll update it now while we have the chance.